services/order.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
```

Comment:
The type hint `list[dict]` is only supported in Python 3.9 and later. Ensure your environment is using a compatible Python version.